### ISSUE TITLE
[Deeplab]  'train_aug' value & unstable loss   local_test.sh, local_test_mobile.sh

### ISSUE THREAD

**Issue by pulse9dev** [tensorflow/models#4854]
### System information
- **What is the top-level directory of the model you are using**: Deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: pip install
- **TensorFlow version (use command below)**: 1.9.0
- **Bazel version (if compiling from source)**: 
- **CUDA/cuDNN version**: 9.0
- **GPU model and memory**: 1080 Ti
- **Exact command to reproduce**:

### Describe the problem
Thanks for sharing amazing code :)

**1. What is meaning of traing_aug=10582 in Pascal dataset.**
        'train': 1464,
        'train_aug': 10582,
        'trainval': 2913,
        'val': 1449,
I found that whole file of 'VOC2012/JPEGImage' is 17125. 
 

**2. When I run 'sh local_tes.sh', loss is unstable between 0.1 to 0.7,**
Loss doesn't converge and decrese at all. even though result from vis isn't bad

Thank you

**Comment by austingg** [comment]
local_test.sh just do a litte test for the repo.  It only finetune 10 iters, the loss is on a batch, which is different for different image. and visualization is used the pretrained model, so the result is good.

**Comment by harshini-gadige** [comment]
@pulse9dev Hi, does this issue still exist ?

**Comment by wt-huang** [comment]
Closing as this is resolved
